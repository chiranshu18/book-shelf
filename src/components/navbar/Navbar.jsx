import React from 'react'
import './navbar.css'

const Navbar = () => {
  return (

    <div className='nav'>

        <div> <hr /> </div>

        <div className='heading'>

            <div className='title'>
                <h3>BS</h3>
            </div>

            <div className='sub-title'>
                <h4>The Book Shelf</h4>
            </div>
 
        </div>

        <div> <hr /> </div>

    </div>

  )
}

export default Navbar