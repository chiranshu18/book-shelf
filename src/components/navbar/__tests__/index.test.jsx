import Navbar from '../Navbar'
import { render , screen} from "@testing-library/react";

describe('navbar', () => {
    
    it('should display heading when rendered', () => {
        render(<Navbar />);
        expect(screen.getByText('BS')).toBeTruthy();
    });

    it('Should render correctly', () => {
        const {asFragment} = render(<Navbar />)
        expect(asFragment()).toMatchSnapshot();
    })
})