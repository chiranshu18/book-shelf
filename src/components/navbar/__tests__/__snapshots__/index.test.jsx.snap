// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`navbar Should render correctly 1`] = `
<DocumentFragment>
  <div
    class="nav"
  >
    <div>
       
      <hr />
       
    </div>
    <div
      class="heading"
    >
      <div
        class="title"
      >
        <h3>
          BS
        </h3>
      </div>
      <div
        class="sub-title"
      >
        <h4>
          The Book Shelf
        </h4>
      </div>
    </div>
    <div>
       
      <hr />
       
    </div>
  </div>
</DocumentFragment>
`;

exports[`navbar Should render correctly 2`] = `
<DocumentFragment>
  <div
    class="nav"
  >
    <div>
       
      <hr />
       
    </div>
    <div
      class="heading"
    >
      <div
        class="title"
      >
        <h3>
          BS
        </h3>
      </div>
      <div
        class="sub-title"
      >
        <h4>
          The Book Shelf
        </h4>
      </div>
    </div>
    <div>
       
      <hr />
       
    </div>
  </div>
</DocumentFragment>
`;

exports[`navbar should display 1`] = `
<DocumentFragment>
  <div
    class="nav"
  >
    <div>
       
      <hr />
       
    </div>
    <div
      class="heading"
    >
      <div
        class="title"
      >
        <h3>
          BS
        </h3>
      </div>
      <div
        class="sub-title"
      >
        <h4>
          The Book Shelf
        </h4>
      </div>
    </div>
    <div>
       
      <hr />
       
    </div>
  </div>
</DocumentFragment>
`;

exports[`navbar should display heading when rendered 1`] = `
<DocumentFragment>
  <div
    class="nav"
  >
    <div>
       
      <hr />
       
    </div>
    <div
      class="heading"
    >
      <div
        class="title"
      >
        <h3>
          BS
        </h3>
      </div>
      <div
        class="sub-title"
      >
        <h4>
          The Book Shelf
        </h4>
      </div>
    </div>
    <div>
       
      <hr />
       
    </div>
  </div>
</DocumentFragment>
`;
